src/Symfony/Component/AssetMapper/ImportMap/ImportMapManager.php

```diff
@@ -337,7 +337,22 @@ private function writeImportMapConfig(array $entries): void
         }
 
         $map = class_exists(VarExporter::class) ? VarExporter::export($importMapConfig) : var_export($importMapConfig, true);
-        file_put_contents($this->importMapConfigPath, "<?php\n\nreturn {$map};\n");
+        file_put_contents($this->importMapConfigPath, <<<EOF
+        <?php
```

Comment:
Can you add a blank line below?

Reply:
While changing this file, what about marking it as being generated?

DI does it:

```
* @internal This class has been auto-generated by the Symfony Dependency Injection Component.
```

And the router does it as well:

```
* This file has been auto-generated by the Symfony Routing Component.
```